source/contracts/Controlled.sol

```diff
@@ -37,6 +37,12 @@ contract Controlled is IControlled {
     }
 
     function suicideFunds(address _target) public onlyControllerCaller returns(bool) {
+        // Transfer REP tokens to target
+        ERC20Basic repToken = ERC20Basic(controller.lookup("Reputation"));
```

Comment:
There are many `ReputationToken` contracts and they aren't registered with controller (because we would have no way to sanely name them).  To get the the reputation token you need a branch to start with and then you can call `getReputationToken()` on it to get the address of the reputation token for that branch.  The only way to start from a global state and get to a branch is by starting with the genesis branch and walking the forks.  This is (technically) an unbounded operation so should not be done here.

I think if you want to be able to suicide owned REP then this function will need a new parameter for `Branch`.